networks/particle_net_pf_sv_4_layers.py

```diff
@@ -0,0 +1,39 @@
+import torch
+from utils.nn.model.ParticleNet import ParticleNetTagger
+
+
+def get_model(data_config, **kwargs):
+    conv_params = [
+        (16, (64, 64, 64)),
+        (16, (128, 128, 128)),
+        (16, (256, 256, 256)),
+        (16, (256, 256, 256)),
```

Comment:
should the number of channels for the edgeconv block in the 4th layer be the same?